<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>874</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Composition Software</string>
  </property>
  <widget class="QLineEdit" name="IntroductionLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>351</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Introduction</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Background</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="BackgroundLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>351</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Outroduction</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="OutroductionLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>351</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>226</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Transitions</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TransitionLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>351</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="IntroductionBrowseButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>70</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackgroundBrowseButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>120</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OutroductionBrowseButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>170</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TransitionBrowseButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>220</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Audio</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AudioLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>351</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AudioBrowseButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>270</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RunButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>350</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="WordListTextEdit">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>459</y>
     <width>361</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>490</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Word List</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
